### PR TITLE
product changes to 1up lineup

### PR DESCRIPTION
**Pull request by ziptyze**
## Description

I have removed the v2 option from the original sweet16 product folder. The product was never actually released.

I have added a folder for the new sweet16v2. It is a 4x4 macro pad with per-key rgb, and rgb underlighting, as well as options for two encoders. It uses a pro-micro for the controller, and includes files for using a pro-micro and also the rp2040 based kb2040 from adafruit. Via support is included for both types of controllers.

I have added a folder for the pi60hse. This is a 60% hot swap pcb with rgb underlighting and multiple layout options for the bottom row, split backspace, and split shift.

## Types of Changes

- [x] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/1upkeyboards/pi60_hse/info.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+    "manufacturer": "1upkeyboards",
+    "keyboard_name": "pi60_hse",
+    "maintainer": "ziptyze",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "usb": {
+        "device_ver": "0x0001",
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/info.json:9]:
```suggestion
```

File: keyboards/1upkeyboards/pi60_hse/config.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 10
+
+#define NOP_FUDGE 0.4
+
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/config.h:11]:
```suggestion

```

File: keyboards/1upkeyboards/pi60_hse/info.json [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP8", "GP6", "GP5", "GP14", "GP20", "GP13", "GP12", "GP11", "GP16", "GP17", "GP10", "GP19", "GP18"],
+        "rows": ["GP4", "GP3", "GP2", "GP1", "GP0", "GP21"]
+    },
+    "mouse_key": {
+        "enabled": true
+    },
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/info.json:34]:
```suggestion
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c:1]: Needs licence header

File: keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [3] = LAYOUT_60_ansi(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c:35]:
```suggestion
    )
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,84 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:1]: Licence header

File: keyboards/1upkeyboards/pi60_hse/readme.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+* Keyboard Maintainer: [ziptyze](https://github.com/ziptyze)
+* Hardware Availability: (https://1upkeyboards.com/shop/controllers/1upkeyboards-pi60hse-60-pcb/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make pi60_hse:default
+
+Flashing example for this keyboard:
+
+    make pi60_hse:default:flash
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/readme.md:14]:
```suggestion
    make 1upkeyboards/pi60_hse:default

Flashing example for this keyboard:

    make 1upkeyboards/pi60_hse:default:flash
```

File: keyboards/1upkeyboards/sweet16v2/kb2040/config.h [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+#define MATRIX_ROW_PINS { GP2,  GP9, GP8,  GP7 }
+#define MATRIX_COL_PINS { GP26, GP1, GP18, GP5 }
+
+#define ENCODERS_PAD_A { GP27, GP4 }
+#define ENCODERS_PAD_B { GP28, GP3 }
+
+#define NOP_FUDGE 0.4
+
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/config.h:35]:
```suggestion

```

File: keyboards/1upkeyboards/sweet16v2/kb2040/info.json [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        "encoder_map": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "mouse_key": {
+        "enabled": true
+    },
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/info.json:27]:
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/kb2040/info.json [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+        "vid": "0x6F75"
+    },
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "encoder_map": true,
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/info.json:18]: Cant be enabled at keyboard level
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/kb2040/info.json [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+            { "flags": 4, "matrix": [3, 3], "x": 196, "y": 60 },
+            { "flags": 4, "matrix": [3, 2], "x": 140, "y": 60 },
+            { "flags": 4, "matrix": [3, 1], "x": 84, "y": 60 },
+            { "flags": 2, "x": 28, "y": 47 },
+            { "flags": 4, "matrix": [3, 0], "x": 28, "y": 60 }
+        ]
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x4": {
+            "c_macro": false,
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/info.json:55]:
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/pro_micro/info.json [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+        "cols": ["F7", "D2", "B1", "C6"],
+        "rows": ["D1", "B5", "B4", "E6"]
+    },
+    "matrix_size": {
+        "cols": 4,
+        "rows": 4
+    },
+    "mouse_key": {
+        "enabled": true
+    },
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/pro_micro/info.json:38]:
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/pro_micro/info.json [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+        "vid": "0x6F75"
+    },
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "encoder_map": true,
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/pro_micro/info.json:21]:
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk [hunk 14]
```diff
@@ -0,0 +1,6 @@
+# MCU name
+MCU = RP2040
+BOOTLOADER = rp2040
+BOARD = GENERIC_RP_RP2040
+
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk:5]: Can also go in info.json like pi60_hse
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/kb2040/config.h [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 4
+
+#define DIODE_DIRECTION COL2ROW
+
+#define MATRIX_ROW_PINS { GP2,  GP9, GP8,  GP7 }
+#define MATRIX_COL_PINS { GP26, GP1, GP18, GP5 }
+
+#define ENCODERS_PAD_A { GP27, GP4 }
+#define ENCODERS_PAD_B { GP28, GP3 }
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/kb2040/config.h:32]: Should be moved to info.json to align with the `promicro` target
```suggestion
```

File: keyboards/1upkeyboards/sweet16v2/pro_micro/info.json [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+            { "flags": 4, "matrix": [3, 3], "x": 196, "y": 60 },
+            { "flags": 4, "matrix": [3, 2], "x": 140, "y": 60 },
+            { "flags": 4, "matrix": [3, 1], "x": 84, "y": 60 },
+            { "flags": 2, "x": 28, "y": 47 },
+            { "flags": 4, "matrix": [3, 0], "x": 28, "y": 60 }
+        ]
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x4": {
+            "c_macro": false,
```
Comment (zvecr) [inline keyboards/1upkeyboards/sweet16v2/pro_micro/info.json:80]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk (hunk 14 above)
Comment (ziptyze) [reply keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk:5]: I was having difficulty getting that to work properly when I have options for multiple types of controllers

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk [hunk 17]
```diff
@@ -0,0 +1,3 @@
+# MCU name
+
+WS2812_DRIVER = vendor
```
Comment (fauxpark) [inline keyboards/1upkeyboards/sweet16v2/kb2040/rules.mk:3]:
```suggestion
WS2812_DRIVER = vendor
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/1upkeyboards/pi60_hse/info.json [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+            { "flags": 2, "x": 155, "y": 13 },
+            { "flags": 2, "x": 183, "y": 13 },
+            { "flags": 2, "x": 212, "y": 13 },
+            { "flags": 2, "x": 208, "y": 51 },
+            { "flags": 2, "x": 183, "y": 51 },
+            { "flags": 2, "x": 157, "y": 51 },
+            { "flags": 2, "x": 126, "y": 51 }
+        ]
+    },
+    "layouts": {
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/info.json:52]:
```suggestion
    "community_layouts": [
        "60_ansi_split_bs_rshift"
    ],
    "layouts": {
```

File: keyboards/1upkeyboards/pi60_hse/info.json [hunk 19]
```diff
@@ -0,0 +1,121 @@
...
+        "LAYOUT_60_ansi": {
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/info.json:53]:
```suggestion
        "LAYOUT_60_ansi_split_bs_rshift": {
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    [0] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+         KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS, KC_EQL,  KC_BSPC, KC_GRV,
+         KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,           KC_ENT,
+        KC_LSFT,             KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,   MO(1),
+        KC_LCTL, KC_LGUI, KC_LALT,                          KC_SPC,                               KC_RALT, KC_RGUI,   MO(1), KC_RCTL
+    ),
+
+    [1] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c:29]:
```suggestion
    [1] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+         KC_GRV, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL, KC_TRNS,
+        KC_TRNS, KC_TRNS,   KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_TRNS,
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS,  KC_HOME, KC_PGUP,          KC_TRNS,
+        KC_TRNS,          KC_VOLU, KC_VOLD, KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_END,  KC_PGDN, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS,   MO(2), KC_TRNS, KC_TRNS
+    ),
+
+    [2] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/default/keymap.c:37]:
```suggestion
    [2] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 23]
```diff
@@ -0,0 +1,100 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    [0] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 24]
```diff
@@ -0,0 +1,100 @@
...
+
+    [1] = LAYOUT_60_ansi(
+         KC_GRV, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL, KC_TRNS,
+        KC_TRNS, KC_TRNS,   KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_TRNS,
+        KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS,  KC_HOME, KC_PGUP,          KC_TRNS,
+        KC_TRNS,          KC_VOLU, KC_VOLD, KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_END,  KC_PGDN, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS,   MO(2), KC_TRNS, KC_TRNS
+    ),
+
+    [2] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:37]:
```suggestion
    [2] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 25]
```diff
@@ -0,0 +1,100 @@
...
+    
+    [0] = LAYOUT_60_ansi(
+         KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS, KC_EQL,  KC_BSPC, KC_GRV,
+         KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,           KC_ENT,
+        KC_LSFT,             KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,   MO(1),
+        KC_LCTL, KC_LGUI, KC_LALT,                          KC_SPC,                               KC_RALT, KC_RGUI,   MO(1), KC_RCTL
+    ),
+
+    [1] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:29]:
```suggestion
    [1] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 26]
```diff
@@ -0,0 +1,100 @@
...
+
+    [2] = LAYOUT_60_ansi(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW,KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          RGB_RMOD,RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD, RGB_M_SN,RGB_M_K, RGB_M_X, RGB_M_G, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [3] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:45]:
```suggestion
    [3] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 27]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [4] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:53]:
```suggestion
    [4] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 28]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [5] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:61]:
```suggestion
    [5] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 29]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [6] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:69]:
```suggestion
    [6] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 30]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [7] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:77]:
```suggestion
    [7] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 31]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [8] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:85]:
```suggestion
    [8] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c [hunk 32]
```diff
@@ -0,0 +1,100 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,                         KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [9] = LAYOUT_60_ansi(
```
Comment (zvecr) [inline keyboards/1upkeyboards/pi60_hse/keymaps/via/keymap.c:93]:
```suggestion
    [9] = LAYOUT_60_ansi_split_bs_rshift(
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**